examples/plotting_with_geoplot.py

```diff
@@ -14,75 +14,77 @@
 
 First we'll load in the data using GeoPandas.
 """
-import geopandas
-
-path = geopandas.datasets.get_path('naturalearth_lowres')
-df = geopandas.read_file(path)
-# Add a column we'll use later
-df['gdp_pp'] = df['gdp_md_est'] / df['pop_est']
-
-boroughs = geopandas.read_file(geopandas.datasets.get_path('nybb')).to_crs(epsg='4326')
-injurious_collisions = geopandas.read_file(
-    "https://github.com/ResidentMario/geoplot-data/raw/master/nyc-injurious-collisions.geojson")
+import geopandas as gpd
```

Comment:
```suggestion
import geopandas
```
Can you use import geopandas only for consistency of our documentation? We try to keep it in full throughout the documentation, so I would prefer having examples in a same manner.

Reply:
:+1: